src/imgui/ImGuiUtils.cpp

```diff
@@ -210,6 +211,32 @@ void HelpMarker(const char* desc, float width, ImFont* tooltipFont)
 	}
 }
 
+void HelpMarker(std::function<const char*()> getText, float width, ImFont* tooltipFont)
+{
+	ImGui::TextDisabled(ICON_FA_QUESTION_CIRCLE_O);
+
+	if (ImGui::IsItemHovered())
+	{
+		ImGui::BeginTooltip();
+		ImGui::PushTextWrapPos(width);
+
+		if (tooltipFont)
+		{
+			ImGui::PushFont(tooltipFont);
+		}
+
+		ImGui::TextUnformatted(getText());
```

Comment:
Some follow up questions.  Whenever I leverage the second line, I get a wildly flashing tooltip popup on hover.  I attempted to leverage both string and string_view as return types and that didn't seem to affect this behavior.  Were you suggesting to use string as part of the follow-up instead of string_view?

Reply:
> Were you suggesting to use string as part of the follow-up instead of string_view?

Yes.